plugins-new/fstab/frontend/src/app/fstab/view/fstab/fstab.component.ts

```diff
@@ -0,0 +1,87 @@
+import { Component } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { FileSystem } from './filesystem.type';
+import { Devices, Fstab } from './devices.type';
+import { NotificationService, TranslateService } from '@ngx-ajenti/core';
+import { lastValueFrom } from 'rxjs';
+
+@Component({
+  selector: 'app-fstab',
+  templateUrl: './fstab.component.html',
+  //styleUrls: [ '../style.less' ],
+})
+export class FstabComponent {
+
+  mounts: Array<FileSystem> = [];
+  fstab: Fstab = {filesystems: []};
+  devices: Array<Devices> = [];
+  showDetails: boolean;
+
+
+  constructor (
+    private httpClient: HttpClient,
+    private translate: TranslateService,
+    private notify: NotificationService,
+  ) {
+    this.loadMounts();
+    this.loadFstab();
+    this.showDetails = false;
+  };
+
+  public async loadMounts(): Promise<void> {
+    await lastValueFrom(this.httpClient.get<FileSystem[]>('/api/fstab/mounts')).then((filesystems) => {
```

Comment:
you don't need `.then` in an async function:

```
    try {
        this.mounts = await lastValueFrom(this.httpClient.get<FileSystem[]>('/api/fstab/mounts'))
    } catch (error) {
      console.error('ERROR:', error);
    }
```

but unless you're showing a custom error UI, it's actually better to leave out try/catch entirely and have a more detailed automatic error traceback in the console